Engine/util/library_posix.h

```diff
@@ -48,8 +48,12 @@ class PosixLibrary : BaseLibrary
 
   AGS::Common::String BuildPath(char *path, AGS::Common::String libraryName)
   {
-    AGS::Common::String platformLibraryName = path;
-    platformLibraryName.Append("/lib");
+    AGS::Common::String platformLibraryName = "";
+    if (path) {
+      platformLibraryName = path;
```

Comment:
No worries. I will mimic the current file brace style. I wouldn't mind amending the AGS coding conventions in the future though. :)